ext_emconf.php

```diff
@@ -11,17 +11,17 @@
   'createDirs' => '',
   'clearCacheOnLoad' => 0,
   'version' => '1.0.5',
-  'constraints' =>
+  'constraints' => 
   array (
-    'depends' =>
+    'depends' => 
     array (
       'php' => '5.3.7-7.0.999',
       'typo3' => '6.2.0-8.99.99',
     ),
-    'conflicts' =>
+    'conflicts' => 
     array (
     ),
-    'suggests' =>
+    'suggests' => 
```

Comment:
yes. I use a script which writes this file with a var_export and this is how PHP generates such an array